source/tests/pd/model/test_force_grad.py

```
@@ -0,0 +1,111 @@
+        self.config["training"]["validation_data"]["systems"] = data_file
+        self.system_index = 0
+        self.batch_index = 0
+        self.get_dataset(self.system_index, self.batch_index)
+        self.get_model()
+
+    def get_model(self):
+        self.model = get_model(self.config["model"]).to(env.DEVICE)
+
+    def get_dataset(self, system_index=0, batch_index=0):
+        systems = self.config["training"]["training_data"]["systems"]
+        rcut = self.config["model"]["descriptor"]["rcut"]
```

Ruff

Local variable `rcut` is assigned to but never used

Remove assignment to unused variable `rcut`

(F841)